spring-ai-docs/src/main/antora/modules/ROOT/pages/api/chat-memory.adoc

```diff
@@ -119,22 +120,70 @@ ChatMemory chatMemory = MessageWindowChatMemory.builder()
     .build();
 ----
 
+==== Supported Databases and Dialect Abstraction
+
+Spring AI supports multiple relational databases via a dialect abstraction. The following databases are supported out-of-the-box:
+
+- PostgreSQL
+- MySQL / MariaDB
+- SQL Server
+- HSQLDB
+
+The correct dialect is auto-detected from the JDBC URL. You can extend support for other databases by implementing the `ChatMemoryDialect` interface.
+
 ==== Configuration Properties
 
 [cols="2,5,1",stripes=even]
 |===
 |Property | Description | Default Value
-| `spring.ai.chat.memory.repository.jdbc.initialize-schema` | Whether to initialize the schema on startup. | `true`
+| `spring.ai.chat.memory.jdbc.initialize-schema` | Controls when to initialize the schema. Values: `embedded` (default), `always`, `never`. | `embedded`
```

Comment:
The prefix of the properties is `spring.ai.chat.memory.repository.jdbc`, not `spring.ai.chat.memory.jdbc`

```suggestion
| `spring.ai.chat.memory.repository.jdbc.initialize-schema` | Controls when to initialize the schema. Values: `embedded` (default), `always`, `never`. | `embedded`
```